src/flamenco/runtime/fd_pubkey_utils.c

```diff
@@ -37,15 +37,17 @@ fd_pubkey_create_with_seed( fd_exec_instr_ctx_t const * ctx,
 
 /* https://github.com/anza-xyz/agave/blob/77daab497df191ef485a7ad36ed291c1874596e5/sdk/program/src/pubkey.rs#L578-L625 */
 int
-fd_pubkey_derive_pda( fd_pubkey_t const * program_id, 
-                      ulong               seeds_cnt, 
-                      uchar **            seeds, 
-                      ulong *             seed_szs,
-                      uchar *             bump_seed, 
-                      fd_pubkey_t *       out ) {
-
+fd_pubkey_derive_pda( fd_exec_instr_ctx_t const * ctx,
+                      fd_pubkey_t const *         program_id, 
+                      ulong                       seeds_cnt, 
+                      uchar **                    seeds, 
+                      ulong *                     seed_szs,
+                      uchar *                     bump_seed, 
+                      fd_pubkey_t *               out ) {
+  /* https://github.com/anza-xyz/agave/blob/6ac4fe32e28d8ceb4085072b61fa0c6cb09baac1/sdk/program/src/pubkey.rs#L579-L581 */
   if( seeds_cnt>MAX_SEEDS ) {
```

Comment:
This should prob be something like:
```
seeds_cnt + (bump_seed ? 1 : 0) > MAX_SEEDS
```

In agave bump is added to the vector of seeds, and the total len of the vector shouldn't be bigger than MAX_SEEDS.